src/main/java/screens/task_management/task_creation_screens/event_creation_screens/EventCreationScreen.java

```diff
@@ -98,21 +106,26 @@ public void actionPerformed(ActionEvent evt) {
                 // get the start and end date+times and parse them
                 LocalDateTime startDate = LocalDateTime.parse(date.getText() + "T" + startTime.getText());
                 LocalDateTime endDate = LocalDateTime.parse(date.getText() + "T" + endTime.getText());
+
+                // create the Event
                 eventCreationController.create(title.getText(), valPriority,
                         startDate, endDate, valRecurring, valFrequency);
 
+                // notify user of success and return to main screen
                 showMessageDialog(this, "Event Created Successfully");
-                screenLayout.show(screens, "main");
-            } catch (Exception e) {
+                screenLayout.show(screens, "StudentMain");
```

Comment:
Just one small thing: do task creation screens need to work for both instructors and students?
If so, maybe this screen needs to be initialized with a string for "Student" or "Instructor" from main so that you know whether to go back to the StudentMain or the InstructorMain

Reply:
Hey Tiare, since I made separate Test and Assignment screens for instructors and Events are exclusive to students, I don't think there should be any issues with instructors returning to the student main screen (as long as I connected everything correctly)